src/models/blueprint-cloner.js

```diff
@@ -0,0 +1,84 @@
+import path from 'path';
+import { copySync } from 'fs-extra';
+import walkSync from 'walk-sync';
+
+import { fileExists } from '../util/fs';
+
+const FILES_BLACKLIST = ['.ds_store', '.git', '.gitkeep'];
```

Comment:
We definitely need a `FILES_BLACKLIST`.  I'm just wondering if there's value to making it an rc option that would include what you have here as defaults.

Reply:
It might be worth it 🤔 

If we set the defaults it's always good to let the user add more customization.

But in what other circumstances will we need to blacklist files besides `clone`?